packages/core/test/forEach.spec.ts

```diff
@@ -7,8 +8,11 @@ beforeEach(() => {
   ;(window as any).DD_RUM = {}
   // prevent 'Some of your tests did a full page reload!' issue
   window.onbeforeunload = () => 'stop'
+  // useful to debug monitored functions spec
+  setDebugMode(true)
```

Comment:
FMU, this does not work as intended. Because each `.spec.ts` file creates a bundle with its own dependencies, a given `.spec.ts` file will evaluate its own copy of `internalMonitoring.ts`, with its own local `monitoringConfiguration` variable. Setting it in the `forEach.spec.ts` won't do anything useful since it doesn't run any test.

Reply:
indeed, I have moved that here without retesting it 🤦 
I'look into it

Reply:
I have tried with a webpack string replacer approach but it messes with the specs that actually test the setDebug behavior or the internal monitoring...
I have reverted all that.